addons/website_slides/views/website_slides_templates_profile.xml

```diff
@@ -17,14 +17,17 @@
                     <t t-if="courses_completed" t-call="website_slides.display_course">
                         <t t-set="courses" t-value="courses_completed"></t>
                     </t>
-                    <p t-else="" class="text-muted">No completed course yet</p>
+                    <div t-else="" class="text-muted d-inline-block">No completed courses yet!</div>
+                    <div class="text-right d-inline-block pull-right">
+                        <a href="/slides/all" class="btn btn-link btn-sm"><i class="fa fa-arrow-right mr-1"/>All Courses</a>
+                    </div>
                 </div>
                 <div class="mb32">
                     <h5 class="border-bottom pb-1">Followed Courses</h5>
                     <t t-if="courses_ongoing" t-call="website_slides.display_course">
                         <t t-set="courses" t-value="courses_ongoing"></t>
                     </t>
-                    <p t-else="" class="text-muted">No followed course yet</p>
+                    <p t-else="" class="text-muted">No followed courses yet!</p>
```

Comment:
Not sure about the plural rule here. I you don't have, you have 0, and 0 is singular. But the rule is unclear.
mmmh.. we should maybe ask to a linguist (@mart-e) !

Reply:
cf the answer at https://english.stackexchange.com/a/26634

    There is no tax cash basis journal defined for this company

but

    There are no public livechat channels to show.

Reply:
You are amazing, doctor @mart-e. Thanks for your expertise!